src/core/forms/forms.py

```diff
@@ -116,6 +116,16 @@ def clean_password_2(self):
         return password_2
 
 
+class GetResetToken(forms.Form):
```

Comment:
Small nitpick, can we add the "Form" suffix to this class name for consistency with the rest of the forms in this module.